### PR TITLE
Documentation: update experimental_apis for v3.1 release

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: Documentation/dev-guide/experimental_apis.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
+For the most part, the etcd project is stable, but we are still moving fast! We believe in the release fast philosophy. We want to get early feedback on features still in development and stabilizing. Thus, there are, and will be more, experimental features and APIs. We plan to improve these features based on the early feedback from the community, or abandon them if there is little interest, in the next few releases. Please do not rely on any experimental features or APIs in production environment.
 
 ## The current experimental API/features are:
 
-- v3 auth API: expect to be stable in 3.1 release
-- etcd gateway: expect to be stable in 3.1 release
+- gateway: beta, to be stable in 3.2 release
+- gRPC proxy: alpha, to be stable in 3.2 release
```
Comment (fanminshi) [inline Documentation/dev-guide/experimental_apis.md]: I wonder if we should have links that point to the experimental api/feature? @xiang90 @heyitsanthony @gyuho

Comment (gyuho) [inline Documentation/dev-guide/experimental_apis.md]: Fixed

**Comment by fanminshi** [comment]
LGTM

**Comment by xiang90** [comment]
lgtm

**Pull request merged**